udnssdk_test.go

```diff
@@ -159,28 +314,305 @@ func Test_Delete_RRSets(t *testing.T) {
 	}
 }
 
-func Test_GetRRSetsPost(t *testing.T) {
-	rrsets, resp, err := testClient.RRSets.GetRRSets(testDomain, testHostname, "ANY")
+func Test_ListAllRRSetsPost(t *testing.T) {
 
-	t.Logf("GetRRSets(%s, %s, \"ANY\")", testDomain, testHostname)
+	if !enableRRSetTests {
+		t.SkipNow()
+
+	}
+	rrsets, err := testClient.RRSets.ListAllRRSets(testDomain, testHostname, "ANY")
+
+	t.Logf("ListAllRRSets(%s, %s, \"ANY\")", testDomain, testHostname)
 	t.Logf("RRSets: %+v\n", rrsets)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func Test_ListTasks(t *testing.T) {
+	tasks, err := testClient.Tasks.ListAllTasks("")
+	t.Logf("Tasks: %+v \n", tasks)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func Test_ListAccountsOfUser(t *testing.T) {
+
+	if !enableAccountTests {
+		t.SkipNow()
+	}
+	accounts, resp, err := testClient.Accounts.GetAccountsOfUser()
+	t.Logf("Accounts: %+v \n", accounts)
+	t.Logf("Response: %+v\n", resp.Response)
+	testAccounts = accounts
+	if err != nil {
+		if resp.Response.StatusCode == 404 {
+			t.SkipNow()
+		}
+		t.Fatal(err)
+	}
+}
+
+/*
+// TODO: Implement Zones
+func TestListZonesOfAccount(t *testing.T) {
+	if testAccounts == nil {
+		t.Logf("No Accounts Present, skipping...")
+		t.SkipNow()
+	}
+	zones, resp, err := testClient.Accounts.GetAccountsOfUser()
+	t.Logf("Zones: %v \n", zones)
+	t.Logf("Response: %+v\n", resp.Response)
+	testZones = zones
+	if err != nil {
+		if resp.Response.StatusCode == 404 {
+			t.SkipNow()
+		}
+		t.Fatal(err)
+	}
+}
+*/
+
+func Test_ListAllDirectionPoolsGeoNoQuery(t *testing.T) {
```

Comment:
Yes.  I’d really like to move all of the tests into ‘process’ specific files.
--
Josh Masseo | Infrastructure Engineer
M: 817-565-6821 |  josh.masseo@ensighten.com

From: Joseph Anthony Pasquale Holsten <notifications@github.com<mailto:notifications@github.com>>
Reply-To: Ensighten/udnssdk <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Wednesday, February 10, 2016 at 12:27 PM
To: Ensighten/udnssdk <udnssdk@noreply.github.com<mailto:udnssdk@noreply.github.com>>
Cc: Josh Masseo <josh.masseo@ensighten.com<mailto:josh.masseo@ensighten.com>>
Subject: Re: [udnssdk] Inf 1816 directional pools (#2)


In udnssdk_test.go<https://github.com/Ensighten/udnssdk/pull/2#discussion_r52501089>:

> +             t.SkipNow()
> +     }
> +     zones, resp, err := testClient.Accounts.GetAccountsOfUser()
> +     t.Logf("Zones: %v \n", zones)
> +     t.Logf("Response: %+v\n", resp.Response)
> +     testZones = zones
> +     if err != nil {
> +             if resp.Response.StatusCode == 404 {
> +                     t.SkipNow()
> +             }
> +             t.Fatal(err)
> +     }
> +}
> +*/
> +
> +func Test_ListAllDirectionPoolsGeoNoQuery(t *testing.T) {


can we pull directional pool tests into their own file?

—
Reply to this email directly or view it on GitHub<https://github.com/Ensighten/udnssdk/pull/2/files#r52501089>.